### DISCUSSION TITLE
Huion 1060P Not detected in OTD

### DISCUSSION THREAD

**Discussion by NayuTW** [OpenTabletDriver/OpenTabletDriver#2121] [Q&A] [answered]
Hello,
I need some help getting OTD to detect my tablet. I am on a windows device and have installed Zadig's WinUSB to interface 1. However when I open the OpenTabletDriver.UX.WPF.exe it says tablet not detected. Im very lost at this point and am open to any help.
Thanks

**Comment by jamesbt365** [comment, marked as answer]
Help > export diagnostic at the top of otd, save that file as a txt and send it here, i'll check whats incorrect.

**Reply by NayuTW** [reply]
[Diagnostics.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/8238599/Diagnostics.txt)

**Reply by jamesbt365** [reply]
https://github.com/X9VoiD/WinUSBCleanup/releases/tag/v1.0

**Reply by NayuTW** [reply]
What do you recommend I do after running the cleanup program and replugging my tablet, because afterwards it still has no tablet detected. Should I redo all of the steps of the installation for OTD?

**Reply by jamesbt365** [reply]
Export another diagnostic.

**Reply by NayuTW** [reply]
[Diag2.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/8238748/Diag2.txt)

**Reply by jamesbt365** [reply]
Your tablet is not getting picked up/ isn't making hid interfaces.
Go into device manager > view > device by container. find your tablet > right click, uninstall device, replug your tablet, if it still doesn't detect reboot your pc

**Reply by NayuTW** [reply]
I rebooted my PC, and the tablet shows up in the Device Manager, however it still does not get detected in OTD.

**Reply by jamesbt365** [reply]
export another diagnostic, hopefully this time it is creating interfaces.

**Reply by NayuTW** [reply]
[Diag3.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/8238759/Diag3.txt)

**Reply by jamesbt365** [reply]
Probably better to come down to discord at this point, could be winusb but its also only showing one interface.

**Reply by NayuTW** [reply]
As a quick update I just went for a clean install and now it works, but only when I run OTD as admin, which I know is not encouraged. Weirdly enough when I used my old huion drivers they never detected unless I ran it as admin too. Not sure what the deal is but im satisfied as long as it works. Thanks for your help.
